src/core_atmosphere/Registry.xml

```diff
@@ -1971,6 +2051,22 @@
                      description="rain evaporation rate"
                      packages="mp_thompson_in"/>
 
+                <var name="vqr" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1}"
+                     description="mass-weighted mean velocity for rain"
+                     packages="mp_thompson_in"/>
+
+                <var name="vqi" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1}"
+                     description="mass-weighted mean velocity for cloud ice"
+                     packages="mp_thompson_in"/>
+
+                <var name="vqs" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1}"
+                     description="mass-weighted mean velocity for snow"
+                     packages="mp_thompson_in"/>
+
+                <var name="vqg" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1}"
+                     description="mass-weighted mean velocity for graupel"
+                     packages="mp_thompson_in"/>
+
```

Comment:
The package attached to these suggests they are needed by the Thompson scheme, and not Noah.